company_website/static/main_page/partials/_constants.sass

```diff
@@ -6,6 +6,7 @@ $orange-color: #FFC03C
 $blue-color: #3777BA
 $hovered-blue-color: #528ac4
 
+$grey-hr-color: rgba(223, 235, 229, 1)
```

Comment:
If you are using `rgba` I assume that you know what is `alfa` channel and you know what you are doing ;)

Reply:
Sure :P It is to give opacity to our color 1 is a default value and we could gave any other valu from 0 to 1 to set the opacity.